static/src/javascripts/projects/common/modules/experiments/ab.js

```diff
@@ -443,9 +418,8 @@ define([
          */
         testCanBeRun: function (test) {
             if (typeof test === 'string') {
-                return testCanBeRun(find(TESTS, function (t) {
-                    return t.id === test;
-                }));
+                var testIndex = TESTS.map(getId).indexOf(test);
+                return testIndex !== -1 && testCanBeRun(TESTS[testIndex]);
```

Comment:
I wouldn't say this indexOf approach is cleaner than `find` (or `filter`), but you could make use of `getTest` here in any case.